docs/browser.md

```diff
@@ -24,6 +24,18 @@ prettier.format("query { }", { parser: "graphql", plugins: prettierPlugins });
 </script>
 ```
 
+### ES6
```

Comment:
Could you change this to say `ES Modules`?

```suggestion
### ES Modules
```

Reply:
@j-f1 sure.